test/modules/standard/Random/bradc/randomAcrossTypes.2-0.good

```diff
@@ -0,0 +1,18 @@
+0.272759 0.192422 0.39718 0.980939 0.0895016 0.957544 0.491363 0.967409
+0.272759i 0.192422i 0.39718i 0.980939i 0.0895016i 0.957544i 0.491363i 0.967409i
+0.272759 + 0.886728i 0.192422 + 0.309948i 0.39718 + 0.816248i 0.980939 + 0.280448i
+0.272759 0.192422 0.39718 0.980939 0.0895016 0.957544 0.491363 0.967409
+0.272759i 0.192422i 0.39718i 0.980939i 0.0895016i 0.957544i 0.491363i 0.967409i
+0.272759 + 0.886728i 0.192422 + 0.309948i 0.39718 + 0.816248i 0.980939 + 0.280448i
+0.272759 0.192422 0.39718 0.980939 0.0895016 0.957544 0.491363 0.967409
+0.272759i 0.192422i 0.39718i 0.980939i 0.0895016i 0.957544i 0.491363i 0.967409i
+0.272759 + 0.886728i 0.192422 + 0.309948i 0.39718 + 0.816248i 0.980939 + 0.280448i
+0.272759 0.192422 0.39718 0.980939 0.0895016 0.957544 0.491363 0.967409
+0.272759i 0.192422i 0.39718i 0.980939i 0.0895016i 0.957544i 0.491363i 0.967409i
+0.272759 + 0.886728i 0.192422 + 0.309948i 0.39718 + 0.816248i 0.980939 + 0.280448i
+0.272759 0.192422 0.39718 0.980939 0.0895016 0.957544 0.491363 0.967409
+0.272759i 0.192422i 0.39718i 0.980939i 0.0895016i 0.957544i 0.491363i 0.967409i
+0.272759 + 0.886728i 0.192422 + 0.309948i 0.39718 + 0.816248i 0.980939 + 0.280448i
+0.272759 0.192422 0.39718 0.980939 0.0895016 0.957544 0.491363 0.967409
+0.272759i 0.192422i 0.39718i 0.980939i 0.0895016i 0.957544i 0.491363i 0.967409i
+0.272759 + 0.886728i 0.192422 + 0.309948i 0.39718 + 0.816248i 0.980939 + 0.280448i
```

Comment:
Oh wait, now I'm seeing a directory-level COMPOPTS, but still don't see how a 2.0.good file would be used...  Maybe this is what you were alluding to as future work in your PR, but it seems a little unfortunate to take this half-step...